api/integrations/rss/rss-bot

```diff
@@ -101,51 +103,59 @@ except OSError:
     print("Unable to store RSS data at %s." % (opts.data_dir,), file=sys.stderr)
     exit(1)
 
-log_file = os.path.join(opts.data_dir, "rss-bot.log")
-log_format = "%(asctime)s: %(message)s"
+log_file = os.path.join(opts.data_dir, "rss-bot.log") # type: str
+log_format = "%(asctime)s: %(message)s" # type: str
 logging.basicConfig(format=log_format)
 
-formatter = logging.Formatter(log_format)
-file_handler = logging.FileHandler(log_file)
+formatter = logging.Formatter(log_format) # type: logging.Formatter
+file_handler = logging.FileHandler(log_file) # type: logging.FileHandler
 file_handler.setFormatter(formatter)
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger(__name__) # type: logging.Logger
 logger.setLevel(logging.DEBUG)
 logger.addHandler(file_handler)
 
 def log_error_and_exit(error):
+    # type: (str) -> None
     logger.error(error)
     logger.error(usage)
     exit(1)
 
 class MLStripper(HTMLParser):
     def __init__(self):
+        # type: () -> None
         self.reset()
-        self.fed = []
+        self.fed = [] # type: List[Any]
 
     def handle_data(self, data):
+        # type: (Text) -> None
         self.fed.append(data)
 
     def get_data(self):
+        # type: () -> Text
         return ''.join(self.fed)
 
 def strip_tags(html):
+    # type: (Text) -> Text
     stripper = MLStripper()
     stripper.feed(html)
     return stripper.get_data()
 
 def compute_entry_hash(entry):
+    # type: (Dict[str, Any]) -> str
     entry_time = entry.get("published", entry.get("updated"))
     entry_id = entry.get("id", entry.get("link"))
     return hashlib.md5(entry_id + str(entry_time)).hexdigest()
 
 def elide_subject(subject):
+    # type: (Text) -> Text
     MAX_TOPIC_LENGTH = 60
     if len(subject) > MAX_TOPIC_LENGTH:
         subject = subject[:MAX_TOPIC_LENGTH - 3].rstrip() + '...'
     return subject
 
 def send_zulip(entry, feed_name):
+    # type: (Any, Text) -> Any
```

Comment:
Isn't `entry` the same variable that's used it `compute_entry_hash`? If so, why does it have [a different type](https://github.com/zulip/zulip/pull/3003/files#diff-b58293a2f31f8a33174c66a8dab0c09aR145)?